ghengin-core/ghengin-core.cabal

```diff
@@ -180,7 +180,7 @@ library backend-independent-bits
                       -- TODO: NOT DEPEND ON VULKAN IN CORE INDEP
                       vulkan,
                       linear-base,
-                      reference-counting,
+                      reference-counting >= 0.2.0.0,
```

Comment:
I accidentally added these. Did you mean to exclude the 0.2 restriction in all but the first build-target? If not, then I can leave it in the PR I suppose.

Reply:
in a cabal package, all versions of a dependency have to be resolved to the same. So you only need to set the bounds e.g. on the library and the same version will have to be picked for the executable and test suite or other sublibraries.

That said, it’s not bad to be explicit! Do keep it like this